dev/Scroller/Scroller.h

```diff
@@ -75,6 +75,11 @@ class Scroller :
     static constexpr int s_zoomFactorChangeMinMs{ 50 };
     static constexpr int s_zoomFactorChangeMaxMs{ 1000 };
 
+    // Number of ticks ellapsed before restarting the Scale animation to allow
+    // the Content rasterization to be triggered after the Idle State is reached
+    // or a zoom factor change operation completed.
+    static constexpr int s_zoomFactorAnimationRestartTicks = 4;
```

Comment:
What is the logic behind the choice of 4?

Reply:
I talked with Keith M. and unfortunately there is no deterministic way for the Scroller to know when the composition layer saw the stoppage to trigger the rasterization. Based on my observations, 4 ticks was always enough.